src/components/region-common.js

```diff
@@ -64,26 +65,38 @@ export const allRegions = [
-  // { value: 'ASIA-EAST2', label: 'asia-east2 (Hong Kong)', locationType: 'region' },
-  // { value: 'ASIA-NORTHEAST1', label: 'asia-northeast1 (Tokyo)', locationType: 'region' },
-  // { value: 'ASIA-NORTHEAST3', label: 'asia-northeast3 (Seoul)', locationType: 'region' },
-  // { value: 'ASIA-SOUTH1', label: 'asia-south1 (Mumbai)', locationType: 'region' },
-  // { value: 'ASIA-SOUTHEAST1', label: 'asia-southeast1 (Singapore)', locationType: 'region' },
-  // { value: 'ASIA-SOUTHEAST2', label: 'asia-southeast2 (Jakarta)', locationType: 'region' },
-  // { value: 'AUSTRALIA-SOUTHEAST1', label: 'australia-southeast1 (Sydney)', locationType: 'region' }
+  { value: 'US-CENTRAL1', label: 'us-central1 (Iowa)', locationType: locationTypes.region },
+  { value: 'NORTHAMERICA-NORTHEAST1', label: 'northamerica-northeast1 (Montreal)', locationType: locationTypes.region },
+  { value: 'SOUTHAMERICA-EAST1', label: 'southamerica-east1 (Sao Paulo)', locationType: locationTypes.region },
+  { value: 'EUROPE-CENTRAL2', label: 'europe-central2 (Warsaw)', locationType: locationTypes.region },
+  { value: 'EUROPE-NORTH1', label: 'europe-north1 (Finland)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST1', label: 'europe-west1 (Belgium)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST2', label: 'europe-west2 (London)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST3', label: 'europe-west3 (Frankfurt)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST4', label: 'europe-west4 (Netherlands)', locationType: locationTypes.region },
+  { value: 'EUROPE-WEST6', label: 'europe-west6 (Zurich)', locationType: locationTypes.region },
+  { value: 'ASIA-EAST1', label: 'asia-east1 (Taiwan)', locationType: locationTypes.region },
+  { value: 'ASIA-EAST2', label: 'asia-east2 (Hong Kong)', locationType: locationTypes.region },
+  { value: 'ASIA-NORTHEAST1', label: 'asia-northeast1 (Tokyo)', locationType: locationTypes.region },
+  { value: 'ASIA-NORTHEAST3', label: 'asia-northeast3 (Seoul)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTH1', label: 'asia-south1 (Mumbai)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTHEAST1', label: 'asia-southeast1 (Singapore)', locationType: locationTypes.region },
+  { value: 'ASIA-SOUTHEAST2', label: 'asia-southeast2 (Jakarta)', locationType: locationTypes.region },
+  { value: 'AUSTRALIA-SOUTHEAST1', label: 'australia-southeast1 (Sydney)', locationType: locationTypes.region }
 ]
+
+// For current phased release of regionality only supporting US and US-CENTRAL1 buckets.
+export const availableBucketRegions = allRegions.slice(0, 2)
+
+// For current phased release of regionality only supporting compute region in US-CENTRAL1
+// and the same region as your workspace bucket
+export const getAvailableComputeRegions = location => {
+  const UsCentralRegion = _.find({ value: 'US-CENTRAL1' }, allRegions)
```

Comment:
Can we avoid magic strings so default locations are defined in a single place?

```suggestion
  const defaultComputeRegion = _.find({ value: defaultComputeRegion }, allRegions)
```

Reply:
My thought was this function explicitly cares that the region is US, and doesn't necessarily care that we've chosen US-CENTRAL1 as the default. 

Its a minor point to me, however. I've made the change.

Reply:
Ah I see. In that case, we could define a `defaultUSComputeRegion` or such but probably overkill in this case.

Reply:
Agreed that would be overkill, but feel free to push back if you feel strongly otherwise.

Reply:
No push back :)